tests/jobs/design2params/test_design2params.py

```diff
@@ -182,6 +182,20 @@ def test_three_column_defaults(tmpdir):
     assert (parsed_defaults.columns == ["keys", "defaults"]).all()
 
 
+@pytest.mark.parametrize("ext, engine", [("xls", "xlwt"), ("xlsx", None)])
```

Comment:
Using `xlwt` as an engine is deprecated according to Pandas docs, so this test will have to be changed at some point.
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_excel.html

Reply:
Yepp, pandas suggests that we use `to_excel` without specifying engine and let them figure out what to use based on extension. However that isn't possible in earlier versions of pandas.

Reply:
The code also supports OpenOffice spreadsheets. We can add that to these tests by the tuple `("ods", "odf")` and adding `odfpy` to `test_requirements.txt`.

Reply:
I'll see how that works with pandas 1.1.5 (might need to catch another error and specify engine again)